source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,106 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v9.1 - Feature Release](#release-v9-1-feature-release)
 - [Release v9.0 - Major Release](#release-v9-0-major-release)
 - [Release v8.1 - Extended Support Release](#release-v8-1-extended-support-release)
 - [Release v8.0 - Major Release](#release-v8-0-major-release)
-- [Release v7.10 - Feature Release](#release-v7-10-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 
+## Release v9.1 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: October 16, 2023**
+
+### Important Upgrade Notes
+ - Improved performance on data retention ``DeleteOrphanedRows`` queries. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for notes on a new migration that was added. Removed feature flag ``DataRetentionConcurrencyEnabled``. Data retention now runs without concurrency in order to avoid any performance degradation. Added a new configuration setting ``DataRetentionSettings.RetentionIdsBatchSize``, which allows admins to to configure how many batches of IDs will be fetched at a time when deleting orphaned reactions. The default value is 100.
+
+**IMPORTANT:** If you upgrade from a release earlier than v9.0, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Compatibility
+ - Updated Chromium minimum supported version to 116+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Cancel** button to the **Delete category** modal.
+ - Added the ability to resize the channel sidebar and right-hand sidebar.
+ - Added two new filtering options (show all channel types and show private channels) to the **Browse channels** modal.
+ - Pre-packaged GitLab plugin version [v1.7.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.7.0).
+ - Pre-packaged Calls version v0.19.1.
+ - Added additional reaction options when viewing threads or messages when the sidebar is larger than its minimum width.
+ - Added a link to [notification documentation](https://docs.mattermost.com/preferences/manage-your-notifications.html) in the **Notification Settings** modal.
+ - Updated the post textbox measurement code to be more reliable.
+ - The ``/invite`` slash command now supports custom user groups.
+ - Re-enabled the remote marketplace functionality, when configured as per ``PluginSettings.EnableRemoteMarketplace`` [documentation](https://docs.mattermost.com/configure/plugins-configuration-settings.html#plugins-enableremotemarketplace).
+ - Added the ability to convert a group message channel to a private channel.
+ - Group message channels now behave like direct message channels, showing a new mention for every new message.
```

Comment:
**Change to**:
Group messages (GMs) now behave like direct messages (DMs). The badge count increases for every new message.

**Context**:
- Recommend we use this opportunity to clearly differentiate channels vs DMs/GMs from how we describe it to users. 
- Also, let's move away from "mention badge", instead opting for "badge". For example, DMs trigger a notification (aka "trigger a mention") but the user didn't explicitly use the @ mention.
- There's [efforts](https://docs.google.com/document/d/1h3Dygq0cpZ8eGfCItIqf_5clhXuhkoHRP4mSeIY1-RI/edit#heading=h.fqqw3aagrxyi) underway re: updating naming

cc: @cwarnermm in case you have more input